### PR TITLE
fix: blazor links

### PR DESCRIPTION
**Pull request by ElderJames**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution

这个PR主要是希望 Ant Design 团队能更换 Blazor 实现的链接。
我们的项目 (https://github.com/ant-design-blazor/ant-design-blazor) 创建于去年底，一直在社区中宣传，到目前已经发展成一个300多人的社区，5人开发团队，10个Contributior，还获得了700+ star 

我还发了两篇文章，表达了我对这个项目的愿景和开发过程：
[如何用 Blazor 实现 Ant Design 组件库？](https://zhuanlan.zhihu.com/p/115046252)
[如何用 Blazor 实现 Ant Design 组件库（二）](https://zhuanlan.zhihu.com/p/125678315)

B站也直播过一些 Blazor 课程：
https://space.bilibili.com/495324682/

本来以为等组件库开发完善再联系官方获得支持和认可，再请求官网添加链接的。今天我才从社区的朋友那得知，最近被另一个项目的作者添加了Blazor实现的链接。

另一个项目是上个月开始的，有很多地方都参考了我们的项目，而且是个人在开发。虽然文档UI接近官方，但是我们项目也是有在做这方面的改进的。

所以希望可以接受这个PR。

image

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️ 

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered docs/spec/introduce.en-US.md](https://github.com/ElderJames/ant-design/blob/master/docs/spec/introduce.en-US.md)
[View rendered docs/spec/introduce.zh-CN.md](https://github.com/ElderJames/ant-design/blob/master/docs/spec/introduce.zh-CN.md)

### CODE REVIEW

**Comment by ram16g** [comment]
支持，这个仓库开发了一段时间了，组件完成度也比原来那个好很多，而且社区已经建设起来了

**Comment by mutouzdl** [comment]
支持，目前为止最完善的Blazor版本

**Comment by hw72869951** [comment]
支持

**Comment by afc163** [comment]
cc @vertonghenb

**Comment by lison666** [comment]
支持，目前为止最完善的Blazor版本

**Comment by wanbolantian** [comment]
支持，j大很早就在推广了，被人截胡很在爽啊

**Comment by vertonghenb** [comment]
@afc163 I understand the confusion, but there are some issues with the ElderJames version of the library which we tried to address in the their repository but was not addressed namely:

1. The repository fails to communicate **internationally**, e.g. English. Which makes it impossible to work together. At append-it/ant-design-blazor we do. For example: https://github.com/Append-IT/ant-design-blazor/issues/23 and we even translate issues if others don't understand (e.g. Chinees).
2. Starting today there will be **3 developers working full-time on the project**, which will result in **15-25 new components by the first of may 2020**. 
3. The architecture of Append-it/ant-design-blazor is far superior then the one being linked here. We believe that putting **C# as a first-class citizen** is  a **must** instead of JavaScript.
4. There are currently a few more components in de elderjames library but they're not state-of-the-art. It's full of subtle bugs and has **no quality standard**.

Therefore we believe this issue can be closed and invite the few developers from the ElderJames library to join us at https://github.com/Append-IT/ant-design-blazor to create an awesome production ready component library based on Ant Design!

**Comment by Loongle** [comment]
@vertonghenb Your suggestion is very good.But James did a lot of publicity in the community.Switching libraries will lead to a crisis of confidence, and I think James can absorb the Suggestions and continue to improve.

**Comment by alexinea** [comment]
I can’t believe that ElderJames team’s project will be “unable to communicate internationally ”. In fact, Ant Design is a project from China, and its successful implementation proves the international collaboration ability of the Chinese team. At present, there is no evidence that ElderJames' team "does not have the ability to collaborate internationally."

In addition, the work of the ElderJames team on Ant Design Blazor (including promotion and development) has already begun. As can be seen from the total number of "stars", the ElderJames team's project is a very successful project with relatively high completion and more recognition.

For now, I think that the Ant Design Blazor implementation of the ElderJames team is the most sensible choice.

**Comment by Loongle** [comment]
@alexinea You are right!

**Comment by hw72869951** [comment]
@vertonghenb您的建议虽然不错，但是我个人认为目前阶段不合适，甚至无益，有害于ant designer blazor  生态的推广，
1、做组件容易，但是生态却很很难，尤其是国内的生态更难，目前中国也只认ANT，请问@vertonghenb你能以4人之力来保证ant designer blazor中国的生态，说国际化目前我觉得太远了(路都不会走就要学会跑步，容易跌跟头)，国际上blazor开源组件也不少，然而中国人却无人问津，就是因为国内缺少系列完整生态教程支持等方面的原因，如果ant design blazor后期因为没有生态或者后续技术支持不到位导致项目流产是我们不愿意看到的
2.	ElderJames在国内具有完整的生态，个人还是希望eleerJames的链接为好，在国内把生态站稳了，再走国际化比较稳妥；或者说vertonghenb能够确保中国的支持

**Comment by vertonghenb** [comment]
> @vertonghenb您的建议虽然不错，但是我个人认为目前阶段不合适，甚至无益，有害于ant designer blazor 生态的推广，
> 1、做组件容易，但是生态却很很难，尤其是国内的生态更难，目前中国也只认ANT，请问@vertonghenb你能以4人之力来保证ant designer blazor中国的生态，说国际化目前我觉得太远了(路都不会走就要学会跑步，容易跌跟头)，国际上blazor开源组件也不少，然而中国人却无人问津，就是因为国内缺少系列完整生态教程支持等方面的原因，如果ant design blazor后期因为没有生态或者后续技术支持不到位导致项目流产是我们不愿意看到的
> 2. ElderJames在国内具有完整的生态，个人还是希望eleerJames的链接为好，在国内把生态站稳了，再走国际化比较稳妥；或者说vertonghenb能够确保中国的支持

Translated:
vertonghenb While your suggestions are good, I personally think they are inappropriate or even unhelpful at this stage and harmful to the promotion of the ant designer blazor eco
1、It is easy to make components, but ecology is very difficult, especially the domestic ecology is even more difficult, at present China only recognizes ANT, please ask @vertonghenb you can guarantee the ecology of ant designer blazor China with the strength of 4 people, saying that internationalization is too far away (the road will not even go to learn to run, easy to fall behind), international blazor open source components are also many, but no one in China is interested, because of the lack of domestic series of complete ecological tutorial support and other reasons, if ant design blazor late because there is no ecology or follow-up technical support is not in place to cause project abortion is what we do not want to see!
2. ElderJames has a complete ecology in China, I still hope that the link of ElderJames is better, it's safer to go international after the ecology is stable in China; or vertonghenb can ensure the support of China.
Translated with www.DeepL.com/Translator (free version)

--- 

I do believe that the library should be made **international by default**, so we will provide multi language support however, English should be the primary language to make something truly international.  

Apart from that, we believe that some design decisions were made recklessly and harms the ecosystem in general. The amount of components simply doesn't count, since half of the repository mentioned here don't work as specified by the official Ant Design documentation which we follow 100% nor are the components documented/unit tested. 

We do believe that the state for further development is still in the Append-IT/ant-design-blazor repository and we invite you to come and help us create an amazing component library.

**Comment by EdiWang** [comment]
Hi @vertonghenb 

## Language

I don't know what's your native language. Although I agree that English should be the most "international" language of choice for projects on GitHub, using Chinese is not doing something wrong, it's not a crime. In China, not everyone can get a chance to stay in a English friendly context to learn and use English. It's not our fault, and we won't be able to change that nature. There are a lot other projects on GitHub that are using all kinds of languages, I don't think Chinese should be especially marked as something special. 

For ANTD, it's originally a Chinese project, so the communities around the project can be very naturally Chinese first. As for the "international" thing, as long as the project is good, it can accept international requests and contributions. 

As @hw72869951 mentioned. The .NET ecosystem in China is very poor comparing to the west world, this is why we as the very few .NET developers in China care so much about our own .NET projects. Especially for projects that put native users at first like @ElderJames 's version of ant blazor. Many many "International" project just don't have a first class Chinese support which makes it hard for us to use in local projects. 

So, whether to use English or to make the project international is entirely up to the project owner. 

## Contribution suggestion

Legally MIT license doesn't ban anyone from using the project and make their own version, however if you think @ElderJames 's project has problem and want to make contributions like adding "international" support or fundamental architecture change, **wouldn't it be nice if your team can just submit PR or join @ElderJames team, or even create a new GitHub organization together?** So that one day you can have a unified project website and link. 

I can see that your team made a lot effort on reconstructing the ant-design-blazor so that it can reach more people with world class communities. But making a own project seems like trying to kick @ElderJames out of the game to us.

**Comment by jijiechen** [comment]
Well, internationalization is really not a big deal at least at this very early stage. As you can see, we can all work in English and will be helping @ElderJames with the project.

Feature completeness and community development are the priorities when we measure an open source project. Frankly, James's project is much better in these dimensions as of now. It's your own idea that your project will be better. We'll have to see if it can come true for others some day. Good luck.

@ElderJames is humble, but people around his project is passionated.

**Comment by vertonghenb** [comment]
Thank you for your reply @EdiWang ! 

### Language
First of all I like the idea that everyone can speak his/her's mother tongue however it makes it unbelievably hard to contribute to open source projects, the language barrier is simply there.
We are planning to translate every issue in foreign languages to English, which makes it possible for more people to read and understand the issue. 

### Contribution Suggestions
First of all we did not know that the project existed so the development was already split up from the start (that's why we added the library to the Official Ant Design website). We even contributed to the @ElderJames library (see: the Alert and Badge component). But we noticed a fundamental flaw in design decisions which we addressed. 

Blazor is not JavaScript first and should be  treated different. Apart from that we also noticed that the library is rather quantity above quality which is a fundamental problem to some of our members, which is up to the project's maintainers to keep track off. 

### Final notes
We are not kicking anyone out of the game, as I said a few times now, we would love to contribute together and have maintainers from around the globe which can help to create an amazing component library and apart from that, learn from each other. We would even like that all the maintainers of the @ElderJames library come and join the open source project! 

### Future
We'll be going further with our development and the schedule for our next release is being prepared as we speak. Before the end of april we'll be having 15-25 industry standard components based on the official docs from Ant Design and I hope we can set these differences aside so we can start doing what we all love doing.

**Comment by xrkffgg** [comment]
看了一下大概了解了。
虽然我对 Blazor 不太懂，但还是能感受 @ElderJames 以及他们团队的热情和信心。

---

After reading, I probably understood.
Although I don’t know much about Blazor, I still feel the enthusiasm and confidence of @ElderJames and their team.

**Comment by vertonghenb** [comment]
### Status update:
I reached out to @ElderJames on Gitter, to see if we can come to an agreement on this.

**Comment by ElderJames** [comment]
Hi, @vertonghenb .

I'm sorry I've been a bit busy lately, but I'm really sorry about this.

>There’s no need to have 2 libraries...

I'm glad you finally realized that.I created this project a long time ago.I still remember that we accepted your pull request very kindly and we are very appreciative for it.We always welcome developers from any country or region to participate.

I also posted the latest news of this project on twitter, which was [reposted](https://twitter.com/shunjiey/status/1240115886469464069) by @SteveSandersonMS and other international friends.We also started slack, both issue and code review in repo are in English and I don't find any problems with internationalization.

Project maintenance is an evolving process.In the early development phase of our project, it took a long time to verify the feasibility of implementing ant design with Blazor. Low code quality is normal, as is the large number of readme in your project and we would love to quickly become an officially recognized ant design implementation.

You mentioned the code specification before, thank you very much! We're already working on improvements, but we just wanted to bring blazor closer to the front-end framework, with some of its specifications.

ant-design-blazor has gained the attention and even participation of many developers since its development. As the project gets larger, we will put more effort into quality and development processes. We've done the same specification on the commit message.There is also preview site, which has plans for unit testing and integration testing.

Ant Design was first developed in China and has been recognized and liked by a large number of developers and enterprise users, which is why our project has received so much attention.We want to work with ant design to bring this design language to the world.

Modesty and comity are fine traditions of the Chinese nation, which we remember from childhood even though they are not very international.My project didn't plan to add a link to the official website so early, but now that yours has appeared, I have to fight for it.

**Comment by JeffreySu** [comment]
First,
I think any official recommendation should always have a fundamental value, which is not only about the project, but also about the person: whether the person can really bring positive energy to the community, whether the person has an open mind, whether the person has the spirit of open source collaboration, whether the person has the potential to learn and improve. Our community needs people who truly serve others, whether it's one person or two or more. I believe that at least @ElderJames is one of them, he has the power to support the project well, and I would be honored if he would let me be a team member to improve the project.

Secondly,
The open source community should be the home of the open and of those of us who are "incomplete." we need to be inclusive, not discriminatory. If someone doesn't want to see flaws or bugs, that person shouldn't even use .NET, he/she should use the best language in the world: *H*.

**Comment by Brian-Ding** [comment]
> First of all we did not know that the project existed so the development was already split up from the start (that's why we added the library to the Official Ant Design website)

How could you say that you don't know James's project existed when you submitted your push request to his repository on March 22, 2020 before you initiated your project on the next day? What a coincidence. 

image

image

And that is definitely not the reason you try to hold the "official link" in a rush and pretend to be a officially recognized and recommended project.

You demo site might have a closer look with the official site, but remember, open source is not about demo/coding only, but also a healthy community. And how can you assure that you have a proper way to do that? At lease, now in the Chinese community, you seems lost your chance. From what I've known, @ElderJames had made a huge effort on the community, both in China and in the globe, and encouraged other developers to write technical blogs as well. As a result, more and more developers are getting to know the .NET Blazor framework and they have been working on Blazor to develop all kinds of project like Raspberry monitor app, enterprise admin app. It is the community initiated by @ElderJames that allows us to code together, to learn together, and to improve and progress together.

Coding is not fighting, like you said, there is no need to have 2 Blazor libraries, so why don't you keep submitting your brilliant code to James's repository instead of creating another one later on and holding the official link by yourselves? I believe James and the community will be more than welcome to have you and your full time development team joined.

**Comment by ElderJames** [comment]
Thank you all for your understanding and support @alexinea @EdiWang @jijiechen @JeffreySu @Brian-Ding

We were happy and determined to build a more open source community, and any meaningful suggestions and PRs are welcome.

Blazor WebAssembly is currently in its infancy, and our work is focused on enriching the Blazor ecosystem, building a library of components available for enterprise production based on the excellent Ant Design Design system. In the process, more developers will be interested and more developers and enterprises will be attracted to use Blazor.

On the other hand, it makes sense for Ant Design to incorporate our Blazor technology and community.  They are happy to see the design system expand and gain more popularity. We are confident that we can make ant-design-blazor a perfect Ant Design implementation and keep their design dream alive.

**Comment by afc163** [comment]
Sorry for reply this issue so late. I am not intented to get involved in this blazor community issue. And I found that merging #23579 was too soon and too rash, both projects were not mature enough to show in header and footer. I will reserve both projects in https://ant.design/docs/spec/introduce#Front-end-Implementation and let time works. Thx for all passion in this issue and let's make Ant Design open source community better.

File: site/theme/template/Layout/Header/More.tsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ export function getEcosystemGroup({ isZhCN }: SharedProps): React.ReactElement {
       </Menu.Item>
       <Menu.Item key="blazor">
         <a
-          href="https://append-it.github.io/ant-design-blazor/"
+          href="https://ant-design-blazor.github.io/"
```
Comment (yoyo837) [inline site/theme/template/Layout/Header/More.tsx:42]: 这里！

File: docs/spec/introduce.en-US.md [hunk 2]
```diff
@@ -63,11 +63,6 @@ const LinksList = () => (
         San UI Toolkit for Ant Design<LinkIcon />
       </a>
     </li>
-    <li>
```
Comment (yoyo837) [inline docs/spec/introduce.en-US.md:66]: 需要删除https://github.com/FE-Driver/vue-beauty 吗？zh-CN.md 没删

File: docs/spec/introduce.zh-CN.md [hunk 3]
```diff
@@ -63,11 +63,6 @@ const LinksList = () => (
         San UI Toolkit for Ant Design<LinkIcon />
       </a>
     </li>
-    <li>
-      <a href="https://github.com/FE-Driver/vue-beauty" target="_blank">
-        vue-beauty (vue)<LinkIcon />
-      </a>
-    </li>
```
Comment (afc163) [inline docs/spec/introduce.zh-CN.md:70]: I will keep looking at these projects and remove inactive projects once a time.

**Pull request merged**